.all-contributorsrc

```diff
@@ -1431,6 +1431,15 @@
         "code"
       ]
     },
+    {
+      "login": "rakshitha123",
+      "name": "Rakshitha Godahewa",
+      "avatar_url": "https://avatars.githubusercontent.com/u/7654679?v=4",
+      "profile": "https://github.com/rakshitha123",
+      "contributions": [
+        "code"
```

Comment:
you haven't contributed any code yet! (that's still to come 😃 )

Fixing typo in docs is a `"docs"` contribution.

Reply:
Changed the contribution type to docs